airflow/jobs/local_task_job.py

```diff
@@ -57,6 +62,7 @@ def __init__(
         self.pool = pool
         self.pickle_id = pickle_id
         self.mark_success = mark_success
+        self.task_runner = get_task_runner(self)
```

Comment:
May not be impotrant, but that is calling this earlier. Should we delay this to in execute() or do you think it's okay to call this earlier?

Reply:
I think because of the code in BaseTaskRunner this is probably better left where it was (BaseTaskRunner can create files, and create a tmp config file etc.)

Reply:
I will revert this then and left `self.task_runner = None` in init.